src/MicrosoftTeams.ts

```diff
@@ -1566,4 +1570,73 @@ namespace microsoftTeams {
       args: args || []
     };
   }
+
+  export interface TaskInfo {
+    /**
+     * The url to be rendered in the webview/iframe.
+     */
+    url?: string;
+
+    /**
+     * JSON defining an adaptive card.
+     */
+    card?: string;
+
+    /**
+     * The requested height of the webview/iframe in pixels.
+     */
+    height?: string;
+
+    /**
+     * The requested width of the webview/iframe in pixels.
+     */
+    width?: string;
+
+    /**
+     * Title of the task module.
+     */
+    title?: string;
+
+    /**
+     * If client doesnt support the URL, the URL that needs to be opened in the browser.
+     */
+    fallbackUrl?: string;
+  }
+
+  /**
+   * Namespace to interact with the task module-specific part of the SDK.
+   * This object is usable only on the content frame.
+   */
+  export namespace tasks {
+    /**
+     * Allows an app to open the task module.
+     * @param taskInfo An object containing the parameters of the task module
+     * @param completionHandler Handler to call when the task module is completed
+     */
+    export function startTask(
+      taskInfo: TaskInfo,
+      completionHandler?: (err: string, result: string) => void
+    ): void {
+      // Ensure that the tab content is initialized
+      ensureInitialized(frameContexts.content);
+
+      let messageId = sendMessageRequest(parentWindow, "start", [taskInfo]);
+      callbacks[messageId] = completionHandler;
+    }
+
+    /**
+     * Complete the task module.
+     * @param result Contains the result to be sent to the bot or teh app. Typically a JSON object or a serialized version of it
+     * @param appId Helps to validate that the call originates from the same appId as the one that invoked the task module
+     */
+    export function completeTask(
+      result?: string | object,
+      appIds?: string[]
+    ): void {
+      // Ensure that the tab content is initialized
+      ensureInitialized(frameContexts.content);
+
+      sendMessageRequest(parentWindow, "complete", [result, appIds]);
```

Comment:
Please make the message name here "tasks.completeTask" instead of just "complete". That's the pattern we've established in the rest of the SDK and makes it clear which API call corresponds to the message.